sendgrid/constants.py

```diff
@@ -1,3 +1,10 @@
+""" Copyright start
+  Copyright (C) 2008 - 2025 Fortinet Inc.
+  All rights reserved.
```

Comment:
Please update copyright for GitHub:

"""
Copyright start
MIT License
Copyright (c) 2025 Fortinet Inc
Copyright end
"""